pkg/ddevapp/config.go

```diff
@@ -462,6 +461,11 @@ func (app *DdevApp) ValidateConfig() error {
 		return err
 	}
 
+	// Stop here if no config found
+	if err := CheckForMissingProjectFiles(app); err != nil {
+		return nil
```

Comment:
Shouldn't we be returning the error here?

Reply:
Added more comments on this check.